docs/FAQ.md

```diff
@@ -0,0 +1,52 @@
+### Frequently Asked Questions
+
+#### Table of Contents
+* [What is KUDO?](#what-is-kudo)
+* [When would you use KUDO?](#when-would-you-use-kudo)
+* [What is a Framework](#what-is-a-framework)
+* [What is a Frameworkversion?](#what-is-a-frameworkversion)
+* [What is an Instance?](#what-is-an-instance)
+* [What is a Planexecution?](#what-is-a-planexecution)
+* [What is a Parameter?](#what-is-a-parameter)
+* [What is a Trigger?](#what-is-a-trigger)
+* [When I create a Framework, it is automatically creating new CRDs?](#when-i-create-a-framework-it-is-automatically-creating-new-crds)
+* [How does it work from a RBAC perspective?](#how-does-it-work-from-a-rbac-perspective)
+* [Is the dependency model an individual controller per workload?](#is-the-dependency-model-an-individual-controller-per-workload)
+
+#### What is KUDO?
+
+KUDO is an operator we are building that is specifically designed to help provide operations to operators. We want to capture the actions that are required for managing applications in production as part of the deployment for the applications themself. To embed those best practices in the operator CRD. So KUDO is the operator that handles those specs that are provided as declarative CRDs.
+
+#### When would you use KUDO?
+
+You would use KUDO when `kubectl apply -f` isn't quite enough to manage your application. If you are just applying the same YAML on updates you probably won't need the extra business logic KUDO gives you.
+
+#### What is a Framework
+
+`Framework` is the highlevel CRD object.
+
+#### What is a Frameworkversion?
+
+#### What is an Instance?
```

Comment:
Reference the RoadMap for dynamic CRDS. 

Or maybe since @gerred is cruising, we start the documentation on dynamic CRDs since we'll be there soon.

Reply:
We might still be able to use this "Instance" vocab word as an "instantiation of a dynamically generated CRD".  It'll still have meaning, but not be a CRD going forward.